tests/checkpoint/conftest.py

```diff
@@ -304,9 +302,9 @@ def titanic_spark_data_context_with_v013_datasource_with_checkpoints_v1_with_emp
                     - airflow_run_id
     """
 
-    context.test_yaml_config(
-        name="my_datasource", yaml_config=datasource_config, pretty_print=False
-    )
```

Comment:
Tried to replace any instances I could with `yaml.load` so we didn't have the change the test too much but still gained the flexibility to remove `test_yaml_config`